core/o11y/Cargo.toml

```diff
@@ -29,6 +29,7 @@ tracing.workspace = true
 tracing-appender.workspace = true
 tracing-opentelemetry.workspace = true
 tracing-subscriber.workspace = true
+log.workspace = true
```

Comment:
There is already `tracing`  crate, so you don't need to use `log` crate.

P.S. I am surprised that `log` crate is included in the workspace 🤔

Reply:
Yeah, that is true. looking into the usage of `log` dependency, I can see it is used in only two places.